extensions/cornerstone-dicom-seg/src/commandsModule.ts

```diff
@@ -251,6 +251,10 @@ const commandsModule = ({
 
           const { dataset: naturalizedReport } = generatedData;
 
+          if (naturalizedReport.StudyID === "No Study ID"){
```

Comment:
Can you add a comment that says that DCMJS assigns a dummy study id during creation, and this can cause problems, so clearing it out?